benchmark/worker/atomics-wait.js

```diff
@@ -7,6 +7,10 @@ const bench = common.createBenchmark(main, {
 });
 
 function main({ n }) {
+  if (typeof SharedArrayBuffer === 'undefined') {
+    throw new Error('SharedArrayBuffers must be enabled in order to run this benchmark');
```

Comment:
Slightly improve the text and get it down to 80 chars in one change!

```suggestion
    throw new Error('SharedArrayBuffers must be enabled to run this benchmark');
```